go/jsonclient/client_test.go

```diff
@@ -226,7 +234,9 @@ func TestPostAndParse(t *testing.T) {
 }
 
 func TestPostAndParseWithRetry(t *testing.T) {
-	leeway := time.Millisecond * 100
+	// leeway is quite large to accommodate for the RFC1123 Retry-After
+	// test
+	leeway := time.Millisecond * 750
```

Comment:
I'd rather you move leeway into the test struct so that you can extend it to 750ms specifically for the /retry-rfc1134 test case.